### DISCUSSION TITLE
Suggestion - Create multiple buttons within integration

### DISCUSSION THREAD

**Discussion by tretabyte** [hass-chargeamps/hass-chargeamps#69] [Q&A]
Hi,

I would like to suggest a creation of multiple buttons as in start charging (remote start button), stop charging (remote stop button), we already have enable and disable but not remote_start or remote_stop (yes I know we have services but it shoulnd't be easier for the user to turn this into a button?), entity to identify the active current and a slider to change the max_current.
I know we have services, but my skills aren't the greatest to turn this into functional buttons, if anyone has a suggestion on how to do it properly I wouldn't mind testing it.

**Comment by jimmyl88** [comment]
Since Charge amp updated their mobile app, my charging schedules no longer work as before. And therefore I want to control the charging completely via HA, tried creating an automation by turning on / off the switch for the charging port. In the new version of the app you can now stop charging in progress.
